src/main/java/seedu/address/model/task/Task.java

```diff
@@ -36,21 +36,22 @@ public Task(Title title, LocalDateTime deadline, UniqueTagList tags, boolean isM
         } else {
             this.deadline = new SimpleObjectProperty<>();
         }
+        this.lastModifiedTime = new SimpleObjectProperty<>(lastModifiedTime);
     }
 
     /**
      * Every field must be present and not null.
      */
     public Task(Title title, UniqueTagList tags) {
-        this(title, null, tags, false, "");
+        this(title, null, tags, false, "", LocalDateTime.MIN);
```

Comment:
This constructor is only used by tests. I am not sure if the tests will consistently pass/fail if this field is allowed to vary across runs.